docs/adding_packages/README.md

```diff
@@ -3,180 +3,131 @@
-two or more folders, dedicated to different versions, each one with its own `conanfile.py` recipe. In any case, those folders should replicate the
-same structure.
+Once you've successfully built an existing recipe following [developing recipes](../developing_recipes_locally.md) tutorial.
+You are set to being.
 
-### `config.yml`
+Make sure you have:
 
-This file lists the versions and the folders where they are located:
+* Fork and then clone the [conan-center-index](https://github.com/conan-io/conan-center-index) git repository.
+* Make sure you are using a recent [Conan client](https://conan.io/downloads) version, as recipes improve by introducing features of the newer Conan releases.
 
-```yml
-versions:
-  "1.1.0":
-    folder: 1.x.x
-  "1.1.1":
-    folder: 1.x.x
-  "2.0.0":
-    folder: all
-  "2.1.0":
-    folder: all
-```
+The easiest way to start is copying a template from our [`package_templates/`](../package_templates) folder to the [`recipes/`](../../recipes/) folder.
+Rename the new folder following the [project name](conanfile_attributes.md#name) guidelines. Read templates [documentation](../package_templates/README.md)
+to find more information.
 
+Quickly, there's a few items to look at:
 
-### `conandata.yml`
+* Add _only_ the latest version in the [`config.yml`](folders_and_files.md#configyml) and [`conandata.yml`](folders_and_files.md#conandatayml)
+* Make sure to update the [`ConanFile` attributes](conanfile_attributes.md) like `license`, `description`, ect...
 
-This file lists **all the sources that are needed to build the package**: source code, patch files, license files,... any file that will be used by the recipe
-should be listed here. The file is organized into two sections, `sources` and `patches`, each one of them contains the files that are required
-for each version of the library. All the files that are downloaded from the internet should include a checksum, so we can validate that
-they are not changed.
+In ConanCenter, our belief is recipes should always match upstream, in other words, what the original author(s) intended.
 
-A detailed breakdown of all the fields can be found in [conandata_yml_format.md](conandata_yml_format.md). We **strongly** encourage adding the [patch fields](conandata_yml_format.md#patches-fields) to help track where patches come from and what issue they solve.
+* Options should [follow these recommendations](conanfile_attributes.md#options) as well as match the default value used by the upstream project.
+* [Package information](build_and_package.md), libraries, components should match as well. This includes exposing supported build system names.
 
-Inside the `conanfile.py` recipe, this data is available in a `self.conan_data` attribute that can be used as follows:
+Where dependencies are involved, there's no shortcuts, inspect the upstream's build scripts for how they usually consume them. Pick the Conan
```

Comment:
`they` and `them` - unclear who're you referring to in each case, use precise wording

Reply:
```suggestion
Where dependencies are involved, there's no shortcuts, inspect the upstream's build scripts for how they are usually consumed. Pick the Conan
```